plugins/hypervisors/kvm/test/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtUtilitiesHelperTest.java

```diff
@@ -0,0 +1,52 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.cloud.hypervisor.kvm.resource.wrapper;
+
+import java.util.UUID;
+
+import com.cloud.utils.script.Script;
+
+import junit.framework.TestCase;
+
+public class LibvirtUtilitiesHelperTest extends TestCase {
+
+    public void testGenerateUUID() {
+        LibvirtUtilitiesHelper helper = new LibvirtUtilitiesHelper();
+        UUID uuid = UUID.fromString(helper.generateUUIDName());
+        assertEquals(4, uuid.version());
+    }
+
+    public void testSSHKeyPaths() {
+        LibvirtUtilitiesHelper helper = new LibvirtUtilitiesHelper();
+        assertEquals("/root/.ssh", helper.retrieveSshKeysPath());
```

Comment:
this is a unit test and will not 'onky' run on kvm. will it introduce the /tmp on windows issue anew?